e2e/test/config/TestConfig.cs

```diff
@@ -1,14 +1,13 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using Microsoft.VisualStudio.TestTools.UnitTesting;
 using System.Diagnostics.Tracing;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
 
-[assembly: Parallelize(Workers = 1, Scope = ExecutionScope.ClassLevel)]
+[assembly: Parallelize(Workers = 0, Scope = ExecutionScope.ClassLevel)]
```

Comment:
Can you comment the effect of this attribute? Is it to run tests in a given test class serially?

Reply:
Sure, I'll add a comment explaining what that means. Workers = 0 makes the test engine use one worker per available core on the computer running it.